### DISCUSSION TITLE
Labelling samples in ord_plot PCoA

### DISCUSSION THREAD

**Discussion by we32i225** [david-barnett/microViz#156] [Q&A] [answered]
Sorry if this has been asked before
I am plotting a PCoA of my microbiome data using microviz's ord_plot function. Plots look great and easy to use. However, i would like to label each of the samples (circles and triangles) within the PCoA plot so i can see where each one falls. 
Is this possible and if so how?
i have included a picture of my plot in its current state
![image](https://github.com/david-barnett/microViz/assets/142319450/f0cee572-ce71-415d-b608-d06aa15c606d)

thanks!

**Comment by david-barnett** [comment, marked as answer]
Hi, do you mean label each individual point, or each colour/shape group?

If individual points, you should be able to add geom_text, or (maybe better geom_text_repel from ggrepel package) to the plot.
set the label aesthetic e.g. geom_text_repel(aes(label = NAMEOFVARIABLEYOUWANTTOLABELWITH))

This will be pretty crowded though, so you might also want to filter the points you label, e.g. by setting the data argument to a filtering function in the geom to e.g. geom_text(data = function(df) filter(df, variable == "value"), mapping = aes(...))

**Comment by we32i225** [comment]
Hi David
just tried this and it works perfectly! thanks for letting me know about geom_text_repel!

thanks
Will
